html/elements/input/range.json

```diff
@@ -0,0 +1,62 @@
+{
+  "html": {
+    "elements": {
+      "input": {
+        "input-range": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/html/Element/input/range",
+            "support": {
+              "chrome": {
+                "version_added": "4"
+              },
+              "chrome_android": {
+                "version_added": "57",
+                "notes": "Chrome implements the <code>slider-vertical</code> value for the non-standard <code>-webkit-appearance</code> property. You shouldn't use this, since it's proprietary, unless you include appropriate fallbacks for users of other browsers."
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "23",
+                "notes": [
+                  "While the specification says that the range input should be drawn vertically if the height is greater than the width, this behavior is not currently implemented. See these Firefox bugs for more information: <a href='https://bugzil.la/840820'>bug 840820</a> and <a href='https://bugzil.la/981916'>bug 981916</a>.",
+                  "Drawing of hash/tick marks is not yet implemented. See <a href='https://bugzil.la/841942'>bug 841942</a> for more information."
```

Comment:
I'd probably cut ` for more information` here too (i.e., writing only "See bug 841942")

Reply:
As for "vertical-orientation" it might be good to use a subfeature for hash/tick marks, to tell the story consistently for all browsers:

* not supported in Firefox
* supported in Chrome but labels are not supported (there is a note on this that they are supported in Chrome 66 but invisible by default, but I wouldn't get this to work).
* don't know about other browsers. I couldn't get it to work in Safari

Reply:
Yes - makes sense. Due to time constraints, I haven't added those suggestions with this push.